### ISSUE TITLE
feat(vizualization): display only one occurrence of a vulnerability for a target

### ISSUE THREAD

**Issue by psyray** [Security-Tools-Alliance/rengine-ng#124]
### Expected feature

In the visualization graph, and the PDF report (Full and vulnerabilities) the same vulnerabilities, duplicated because they are found on multiple endpoints, are displayed multiple times.

##### Zoom in
![image](https://github.com/Security-Tools-Alliance/rengine-ng/assets/1230954/4eb733a6-f2fa-4686-8b11-8986f56c74ff)

##### Zoom out
![image](https://github.com/Security-Tools-Alliance/rengine-ng/assets/1230954/5e304fd8-e58b-417e-878a-65c6845a2fba)

This make the visualization graph unreadable.

#### Report

![image](https://github.com/Security-Tools-Alliance/rengine-ng/assets/1230954/8edb3c4e-dda5-4056-8156-6a34d05f88df)

So the best way to make it more readable is to display the same vuln only once.
For the visualization, no need to display the endpoints, on the report endpoints could be listed inside the vuln section

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
